backend/functions/lib/adapters/types/trial.attorney.assignment.response.ts

```diff
@@ -0,0 +1,6 @@
+export class TrialAttorneyAssignmentResponse {
```

Comment:
Modify this to a type interface and move into a types file. Similar to cases.d.ts and similar variable naming.

Reply:
Done.